modules/retail/pages/retail-deploy-terminals.adoc

```diff
@@ -114,3 +114,12 @@ image::retail_imagesync_formula.png[scaledwidth=80%]
 
 . Power on your POS terminals.
 . The branch server will bootstrap the terminals according to the data you have provided.
+
+
+== Rate Limiting Terminals
+
+Salt is able to run commands in parallel on a large number of terminals.
+This can potentially create large amounts of load on your infrastructure.
+You can use rate-limiting parameters to control the load in your environment.
```

Comment:
```suggestion
You can use rate limiting parameters to control the load in your environment.
```
Change for consistency.

Reply:
I'm sure we've already had this discussion. I was hyphenating because it's an adjective here ("rate-limiting" modifies "parameters').